lib/periodic_jobs.rb

```diff
@@ -99,8 +99,6 @@
   # Updates Cypress files in vets-website with data from Google Analytics.
   mgr.register('0 13 * * 1', 'Mobile::V0::WeeklyMaintenanceWindowLogger')
   # Weekly logs of maintenance windows
-  mgr.register('0 20 * * *', 'ClaimsApi::ClaimAuditor')
-  # Hourly slack alert of errored claim submissions
   mgr.register('0 * * * *', 'ClaimsApi::ReportHourlyUnsuccessfulSubmissions')
   # Daily alert of pending claims longer than acceptable threshold
```

Comment:
I think these comments got flip flopped in the previous add for the report hourly. if you look at the git blame you can see 103 and 104 belong together according to the previous PR. 103 seems to align with the ReportHourlyUnusuccessful job. 

probably just due to the way the comments are typically _under_ their respective job for some reason?